### PR TITLE
chore(wrapper-gen): use information from CEM (VIV-1578)

### PR DESCRIPTION
**Pull request by RichardHelm**
* All components are marked with `@component`
* All public components are marked with `@public`
* All v-model are declared via `@vueModel`

### CODE REVIEW

File: libs/components/src/lib/appearance-ui/appearance-ui.ts [hunk 1]
```diff
@@ -22,9 +22,7 @@ Appearance.Filled
 
 
 /**
- * Base class for appearance-ui
- *
- * @public
+ * @component appearance-ui
```
Comment (rachelbt) [inline libs/components/src/lib/appearance-ui/appearance-ui.ts:25]: not sure we need this - this is not a component to use :)

File: libs/components/src/lib/data-grid/data-grid.ts [hunk 2]
```diff
@@ -24,9 +24,8 @@ export type ValueOf<T> = T[keyof T];
 export type DataGridSelectionMode = ValueOf<typeof DataGridSelectionMode>;
 
 /**
- * Base class for data-grid
- *
  * @public
+ * @component data-grid
  * @slot - Default slot.
  * @event cell-click - Event that fires when a cell is clicked
```
Comment (rachelbt) [inline libs/components/src/lib/data-grid/data-grid.ts:30]: Are this is needed for Vue as well?
```
ComponentRegister.addComponentOverride('data-grid', component => {
  component.attributes.push({
    name: 'rowsData',
    description: 'Array of objects representing the rows of the grid.',
    type: [{ text: 'any[]', vuePropType: 'Array' }],
  });
});
```

**Review by rachelbt — COMMENTED** [review]
are components that are under lying such as popup, elevation, listbox still need the public @component?
You are missing - range slider

File: libs/components/src/lib/appearance-ui/appearance-ui.ts (hunk 1 above)
Comment (RichardHelm) [reply libs/components/src/lib/appearance-ui/appearance-ui.ts:25]: Yes, the non-public components are hidden by not having `@public`

**Review by TaylorJ76 — APPROVED** [review]

**Pull request merged**